types/extra-types.d.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * This file contains the types that are required for compilation of the
+ * Polymer generated type declarations, but which could not themselves be
+ * automatically generated.
+ */
+
+/// <reference path="lib/mixins/property-effects.d.ts" />
+
+/**
+ * Types from "externs/closure-types.js"
+ */
+
+type Polymer_PropertyEffects = Polymer.PropertyEffects;
```

Comment:
Why is this necessary? Add a comment?

Reply:
No longer necessary. Fixed in https://github.com/Polymer/gen-typescript-declarations/pull/47.